### DISCUSSION TITLE
আপনাদের কি `Dark` থিম নিয়ে কোন চিন্তা আসে ?

### DISCUSSION THREAD

**Discussion by Rafat97** [devsonket/devsonket.github.io#617] [প্রশ্নোত্তর]
আপনাদের  এই  বিষয় নেয়া মতামত কি ?

**Comment by sabbirshawon** [comment]
> আপনাদের এই বিষয় নেয়া মতামত কি ?

সাইটের মধ্যে ডার্ক থিম নিয়ে কথা বলতেছেন নাকি? @Rafat97 ভাই।

**Reply by Rafat97** [reply]
@sabbirshawon  জি ভাই। কারণ, বর্তমান সময়ে বেশির ভাগ ওয়েব-সাইট Dark mode সাপোর্ট করে।

**Comment by 0xMahabub** [comment]
Dark theme is necessary 👍🏻
